op-batcher/batcher/channel_manager.go

```diff
@@ -414,3 +475,26 @@ func (s *channelManager) Close() error {
 	}
 	return nil
 }
+
+// Requeue rebuilds the channel manager state by
+// rewinding blocks back from the channel queue, and setting the defaultCfg.
+func (s *channelManager) Requeue(newCfg ChannelConfig) {
+	newChannelQueue := []*channel{}
+	blocksToRequeueInChannelManager := []*types.Block{}
```

Comment:
I find the `InChannelManager` suffix superfluous and making the variable name unnecessarily long. We are in the context of the channel manager here, so I suggest we shorten it.

Reply:
[a9b9e93](https://github.com/ethereum-optimism/optimism/pull/12002/commits/a9b9e93139866cc9018309ac788aef1cd6904820)